cmd/server/config.go

```diff
@@ -11,9 +10,12 @@ import (
 )
 
 const (
-	ocidRoot            = "/var/lib/ocid"
+	ocidRoot            = "/var/lib/containers"
+	ocidRunRoot         = "/var/run/containers"
 	conmonPath          = "/usr/libexec/ocid/conmon"
-	pausePath           = "/usr/libexec/ocid/pause"
+	pauseImage          = "kubernetes/pause"
+	pauseCommand        = "/pause"
+	defaultTransport    = "docker://"
```

Comment:
this would be implied at some point by the image reference coming from k8s I believe, I'm not sure why we need this in the config